thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/auth/ThirdEyeCredentials.java

```diff
@@ -22,9 +22,18 @@
 import java.util.Objects;
 
 
-public class Credentials {
+public class ThirdEyeCredentials {
   String principal;
   String password;
+  String token;
```

Comment:
Is it mainly for service account?

Reply:
Token exists for all the users. This PR only addresses injecting the authenticated user into the `SecurityContext`. That way we can access the user from every api call.

I will send a followup PR to add the service restriction.